nextcord/guild.py

```diff
@@ -2513,34 +2503,6 @@ async def fetch_roles(self, *, cache: bool = False) -> List[Role]:
 
         return roles
 
-    @overload
-    async def create_role(
```

Comment:
Why remove the overloads?

Reply:
## What is `@overload` ?

### **`@overload` allows describing methods that support multiple different combinations of argument types.**
I saw this from here... 
![image](https://user-images.githubusercontent.com/39167433/164039033-37af194b-c48f-4d3c-8108-ce557aa38bc8.png)

## Why it was there? (from my observation)
- These are two parameters called `color`  and  `colour`.
- These are basically the same, `colour` is an aliase of `color`. 
- For describing both in create_role() method `@overload` was there.

## Why it's not necessary then? 
- Every other argument and return types were exactly the same in those `@overload` create_role() only `color`  and  `colour` changed.
- We can just add those two arguments `color` and `colour` normally (which was already there) and I just added the docstring for `color` which was missing.
- `@overload` is not really useful for this case from my point of view.

Reply:
Doesn't seem to describe why you removed it, overloads can show possible signatures, signatures do not include passing both color and colour

Reply:
For hinting purposes it's better since you can specify `color` and not `colour`, or specify `colour` and not `color`, but you can't specify both.

While it's not mandatory to have, I don't see a reason to remove it.